packages/sitecore-jss/src/personalize/layout-personalizer.test.ts

```diff
@@ -36,140 +38,383 @@ describe('layout-personalizer', () => {
       personalizeLayout(layoutData, variant);
       expect(layoutData.sitecore.context.variantId).to.equal(variant);
     });
+
+    describe('with component variant ids', () => {
+      it('should not return anything', () => {
```

Comment:
please expand the test case conditions description

Reply:
correction: this test block just duplicates the default personalizeLayout tests with no checks related to component variantIds specifically. It's best to have some new tests here.